src/OrbitGl/TrackHeader.cpp

```diff
@@ -119,25 +91,34 @@ void orbit_gl::TrackHeader::DoUpdateLayout() {
 }
 
 void TrackHeader::UpdateCollapseToggle() {
-  const float label_height = layout_->GetTrackTabHeight();
-  const float half_label_height = 0.5f * label_height;
-  const float x0 = GetPos()[0] + layout_->GetTrackTabOffset() +
-                   layout_->GetTrackIndentOffset() * track_->GetIndentationLevel();
-  const float button_offset = layout_->GetCollapseButtonOffset();
+  const float x0 = GetPos()[0] + layout_->GetTrackIndentOffset() * track_->GetIndentationLevel();
   const float size = layout_->GetCollapseButtonSize(track_->GetIndentationLevel());
-  const float toggle_y_pos = GetPos()[1] + half_label_height - size / 2;
-  Vec2 toggle_pos = Vec2(x0 + button_offset, toggle_y_pos);
+
+  constexpr float kOffsetX = -0.5f;
```

Comment:
Could you explain why this offsets with a comment?

Reply:
this was an attempt to fix the asymmetry, but it doesn't work all the time, so I'm removing it.